commands.pm

```diff
@@ -340,24 +342,22 @@ sub start_server {
     $child->autoflush(1);
     $isotovideo->autoflush(1);
 
-    my $pid = fork();
-    die "fork failed" unless defined $pid;
+    my $process = process(sub {
+            $SIG{TERM} = 'DEFAULT';
+            $SIG{INT}  = 'DEFAULT';
+            $SIG{HUP}  = 'DEFAULT';
+            $SIG{CHLD} = 'DEFAULT';
 
-    if ($pid == 0) {
-        $SIG{TERM} = 'DEFAULT';
-        $SIG{INT}  = 'DEFAULT';
-        $SIG{HUP}  = 'DEFAULT';
-        $SIG{CHLD} = 'DEFAULT';
+            close($child);
+            $0 = "$0: commands";
+            run_daemon($port, $isotovideo);
+            Devel::Cover::report() if Devel::Cover->can('report');
+            _exit(0);
+    })->blocking_stop(1)->internal_pipes(0)->set_pipes(0)->start;
 
-        close($child);
-        $0 = "$0: commands";
-        run_daemon($port, $isotovideo);
-        Devel::Cover::report() if Devel::Cover->can('report');
-        _exit(0);
-    }
     close($isotovideo);
-
-    return ($pid, $child);
+    $process->on(collected => sub { print STDERR "commands process exited: " . shift->exit_status . "\n"; });
```

Comment:
Why not use the diag function here?

Reply:
forgot :)

Reply:
Just, checked, also here. we didn't used to import bmwqemu here - see https://github.com/os-autoinst/os-autoinst/pull/1006#discussion_r207459425

Reply:
makes me torn between both xD